tests/atoms/gaia_data_layer.js

```diff
@@ -45,16 +45,17 @@ var GaiaDataLayer = {
 
   disableBluetooth: function() {
     var bluetooth = window.navigator.mozBluetooth;
-    if (bluetooth.enabled) {
+    if (bluetooth.defaultAdapter.state == 'enabled') {
```

Comment:
Ah, right, so if (bluetooth.defaultAdapter.state != 'disabled') { is probably the solution.